sdks/java/extensions/zetasketch/src/test/java/org/apache/beam/sdk/extensions/zetasketch/HllCountTest.java

```diff
@@ -281,12 +283,12 @@ public void testMergePartialGlobally() {
 
   @Test
   @Category(NeedsRunner.class)
-  public void testMergePartialGlobally_MergeWithSketchForEmptySet() {
+  public void testMergePartialGlobally_EmptyInput() {
```

Comment:
emmm...I don't think underscore `_` is a naming convention for non const thing. Maybe consider "With"?

Reply:
Nice catch! I am following the style in the [zetasketch library](https://github.com/google/zetasketch/blob/4aa44e9cb543d766318b919fc6bb359bdf7b0809/javatests/com/google/zetasketch/HyperLogLogPlusPlusTest.java#L97), which I think is a google style? But I am not sure if this complies to Beam style. So I have changed it.